gulpfile.js

```diff
@@ -41,12 +41,7 @@ const html = () => {
 //Filewatcher and live-reload
 export const watchpug = () => {
     browserSync.init({
-        server: {
-            baseDir: "./",
-            open: true,
-            cors: true,
-            notify: false,
-        },
+        server: './features/../',
```

Comment:
@AddMoreScripts why did you update baseDir to './features/../'? I guess it is same as './'.

Reply:
I fixed this setting in the last commit.
In fact, this line is not important, because. the watch parameter is set to false.
I was looking for a way to eliminate the high CPU usage due to the large number of files in the releases and documentation folder.
I changed the gulp.watch setting a bit in the next commit.